source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -327,6 +331,9 @@ Node Details
   * - VerifyLocation
     - May have exactly 1 child node. The prompt will play before verifying the callers location. The child node will be played after verifying the callers location.
 
+  * - DynamicRoute
+    - May have exactly 1 child node. We will evaluate the custom data field value specified on this node's dynamic_route_destination. With non-SIP integration, if the extracted value is valid phone number and we are allowed to connect the call given your settings, we will play the prompt and transfer the call, otherwise the child node will be played without the prompt. When SIP integrated, we allow transferring to a string, the destination should be routable by your SIP infrastructure.
```

Comment:
`is valid phone number` => `is a valid phone number`

How about:
`we are allowed to connect the call` => `the destination phone number is in an allowed region given your settings`

`otherwise the child node will be played` => `otherwise the child node will be executed`

`When SIP integrated, we allow transferring to a string` => `When SIP integrated, we also allow transferring to any string (such as an extension or email address)`

`, the destination` => `, in which case the destination`

Reply:
Much appreciated, updated and force pushed.